File: CSharp/Shop/Ui/Ui.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Shop.Ui {
    using Items;
    using Customers;
    public interface Ui {
        public string MainMenu();
        public string ShopMenu(List<ItemBatch> items);
        public string CustomerMenu(Customer c);
        public void CustomerBuyMenu(List<ItemBatch> items);
    }
}

File: Java/TddShop/src/Ui/Ui.java
package Ui;

import java.util.List;

import Customer.Customer;
import Item.*;
public interface Ui {
    public String MainMenu();
    public String ShopMenu(List<ItemBatch> itemBatches);
    public String CustomerMenu(Customer c);
    public String CustomerBuyMenu(List<ItemBatch> items);
    public String CustomerRefundMenu(List<Item> items);
}
